private/bufpkg/bufconfig/bufconfig.go

```diff
@@ -143,6 +143,35 @@ func WriteConfigWithDependencyModuleReferences(dependencyModuleReferences ...buf
 	}
 }
 
+// WriteConfigWithBreakingConfig returns a new WriteConfigOption that sets the breaking change
+// config for the module.
+//
+// If this option is used and the version used will be consistent with the rest of the config.
+func WriteConfigWithBreakingConfig(breakingConfig *bufbreakingconfig.Config) WriteConfigOption {
+	return func(writeConfigOptions *writeConfigOptions) {
+		writeConfigOptions.breakingConfig = breakingConfig
+	}
+}
+
+// WriteConfigWithLintConfig returns a new WriteConfigOption that sets the lint config for // the module.
+//
+// If this option is used and the version used will be consistent with the rest of the config.
```

Comment:
nit: Similar to above
```suggestion
// If this option is used then the version used must be consistent with the rest of the config.
```